README.md

```diff
@@ -170,6 +170,24 @@ To enable it, set your cusdis name in `_config.yml`:
 cusdis_app_id: my_data-app-id                                     
 ```
 
+#### Utterances
+[Utterances](https://utteranc.es) is a very good alternative to Cusdis and Disqus. It has GitHub login, rather than Google/Twitter login and doesn't ask permission. It is open source and the code is on GitHub. It stores the comments as GitHub issues for a page.
+
+To install Utterances, fill out the utterances lines in config.yml and install the app from [here](https://github.com/apps/utterances) to your repo. If you don't, Utterances will post a comment to install the app.
+
+It also has support for labels, which is exactly GitHub Issue labels. And, it supports emojis.
+
+After installing, fill out these lines under the `comments:` line:
+```yaml
+utterances:
+	repo:
+	issue-term:
+	theme:
+	label:
```

Comment:
Seems like the yaml is wrong here

Reply:
idk, that code always came wrong, will try to fix it within today

Reply:
maybe i will try changing it from yaml to yml

Reply:
changed now